src/log-store/Cargo.toml

```
@@ -5,7 +5,7 @@ edition.workspace = true
 license.workspace = true
 
 [build-dependencies]
-protobuf-build = { version = "0.14", default-features = false, features = [
+protobuf-build = { version = "0.15.1", default-features = false, features = [
```

Does this also work?
```suggestion
protobuf-build = { version = "0.15", default-features = false, features = [
```

---

Yes, this works too.

---

The update of "protoc lookup" happened at 0.14.1, so 0.15 clearly works.